integration-test-example.js

```diff
@@ -0,0 +1,24 @@
+// Integration test file with intentional issues for commenting
+function processUserData(users) {
+    // Missing input validation
+    let results = [];
```

Comment:
♻️ What do you think about adding input validation here? It might prevent runtime errors.